src/main/java/seedu/findvisor/model/util/SampleDataUtil.java

```diff
@@ -57,4 +69,15 @@ public static Set<Tag> getTagSet(String... strings) {
                 .collect(Collectors.toSet());
     }
 
+    /**
+     * Returns a meeting object based on strings given.
+     */
+    public static Optional<Meeting> convertToMeeting(String[] meetingParameters) {
```

Comment:
Why does this function take a `String[]` instead of `LocalDateTime startDateTime, LocalDateTime endDateTime, String meetingRemark`?

Reply:
it takes a String[] so that it is easier to create a meeting where I can use the method with the strings:
{"dateTime1, dateTime2", "meeting Remark} 
 
instead of repeatedly calling: 
`Optional.of(new Meeting(DateTimeUtil.parseDateTimeString("dateTime1"), DateTimeUtil.parseDateTimeString("dateTime2"), "meetingRemark"))`

or

`Optional.of(new Meeting(LocalDateTime.of(dateParameters1), LocalDateTime.of(dateParameters2), "meetingRemark"))`

where each dateParameter consists of the year, month, day, hour, minute.

Reply:
I get where you are coming from, but I'd suggest defining the method as `convertToMeeting(String startDateTimeStr, String endDateTimeStr, String meetingRemark)`, this give you compile time error checking. With the current implementation it's possible to get a runtime error if you pass in an array with invalid size.

Reply:
Alright I understand, will change the implementation